### DISCUSSION TITLE
Any scripts to convert from diffusers format to a single safetensor? to use in comfyui or ldm implementation

### DISCUSSION THREAD

**Discussion by nighting0le01** [huggingface/diffusers#7508] [General]
Hi, just wanted to confirm if we have Any scripts to convert from diffusers format to a single safetensor? to use in comfyui or ldm implementation

**Comment by asomoza** [comment]
Hi, there's this script:

https://github.com/huggingface/diffusers/blob/main/scripts/convert_diffusers_to_original_sdxl.py

Also if you load the model in a pipeline, you can use `pipeline.save_pretrained("model/path")`

**Reply by tolgacangoz** [reply]
Plus, these: [convert_diffusers_sdxl_lora_to_webui.py](https://github.com/huggingface/diffusers/blob/main/scripts/convert_diffusers_sdxl_lora_to_webui.py), [convert_diffusers_to_original_stable_diffusion.py](https://github.com/huggingface/diffusers/blob/main/scripts/convert_diffusers_to_original_stable_diffusion.py).
